src/components/forms/UserFilter/index.tsx

```diff
@@ -60,6 +63,20 @@ export const BaseUserSelectFilter = (props: BaseUserSelectFilterPropTypes) => {
     defaultHandler(option, props);
   };
 
+  useEffect(() => {
+    if (!(defaultUserNameValue === SELECT_USERNAME && ENABLE_DEFAULT_PLAN_USER_FILTER)) {
+      return;
+    }
+
+    const { username } = getUser(store.getState());
+    const option = {
+      label: username,
+      value: '',
```

Comment:
Why is the value empty?

Reply:
its supposed to be the id of the user. which we don't have at that point, fortunately we can and are using the `username` to get the plans assigned to a user, so the userId is optional.

Reply:
if we are using the username to search, would it make sense for the value to be username?

Reply:
the username is also used as the in react-select's dropdown,

Reply:
I mean this eventually creates something similar to HTML options ie `<option value="value">label</option>` right?

So here we'll end up with something like `<option value="">superset-user</option>` - which is weird from a HTML point of view, right?  I am asking is there any harm in having `<option value="superset-user">superset-user</option>`?

Additionally, if as you say we are using the username to filter, does it mean that we filter using the label of the react-select option?

Reply:
>  I am asking is there any harm in having `<option value="superset-user">superset-user</option>`?

I suppose ther is no harm in having the `value` and the `label` being the same value.

Reply:
@p-netm could you confirm what is used when we are doing the filtering?  Coz if it is `label` then I would want it changed if possible (in a different issue) because I prefer to stay as close to HTML standards/principles as possible - in this case that would be that for select options, it is the `value` that is important and submitted to be used, and the `label` is just for display - despite this being js-land where we can do whatever we want.

Reply:
> I suppose ther is no harm in having the value and the label being the same value.

It is actually pretty common for HTML

Reply:
There no any harm

Reply:
@moshthepitt @ciremusyoka Thanks for the review, I will look at this more closely. for the now, I do not hink it will make it for the release.